firebase_admin/messaging.py

```diff
@@ -373,10 +373,11 @@ def batch_callback(_, response, error):
 
         batch = http.BatchHttpRequest(
             callback=batch_callback, batch_uri=_MessagingService.FCM_BATCH_URL)
+        transport = self._build_transport(self._credential)
```

Comment:
Is the reason to use `_build_trasnport` here (instead of directly using `_auth.authorized_http`) so we can write tests easily?

Reply:
Yes.